src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/InMemoryTokenCacheAccessor.cs

```diff
@@ -44,7 +44,7 @@ public InMemoryTokenCacheAccessor(ICoreLogger logger)
         }
 
         #region Add
-        public virtual void SaveAccessToken(MsalAccessTokenCacheItem item)
```

Comment:
Is InMemoryTokenCacheAccessor needed anymore? I don't see it referenced.

Reply:
Yea, removed.